File: Python/Bank_Account_and_Transaction_Ledger/models/transaction.py
from datetime import datetime

class Transaction:
    def __init__(self, account_number, transaction_type, amount):
        self.account_number = account_number
        self.transaction_type = transaction_type
        self.amount = amount
        self.timestamp = datetime.now()
        
    def display(self):
        print(f"[{self.timestamp}] {self.account_number} | {self.transaction_type} | {self.amount}")

File: Bank_Account_and_Transaction_Ledger/Models/Transaction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class Transaction
    {
        public string AccountNumber { get; private set; }
        public string TransactionType { get; private set; }
        public double Amount { get; private set; }
        public DateTime Timestamp { get; private set; }

        public Transaction(string accountNumber, string transactionType, double amount)
        {
            AccountNumber = accountNumber;
            TransactionType = transactionType;
            Amount = amount;
            Timestamp = DateTime.Now;
        }

        public void Display()
        {
            Console.WriteLine($"[{Timestamp}] {AccountNumber} | {TransactionType} | {Amount}");
        }
    }
}
